addon-test-support/audit.js

```diff
@@ -80,7 +87,26 @@ function runA11yAudit(contextSelector = '#ember-testing-container', axeOptions)
 
   return auditPromise
     .then(a11yAuditCallback)
-    .finally(() => document.body.classList.remove('axe-running'));
+    .finally(() => {
+      document.body.classList.remove('axe-running');
+      markPerf('a11y_audit', 'a11y_audit_start', 'a11y_audit_end');
+    });
+}
+
+/**
+ * Adds end performance marker. Measures execution between start and end markers.
+ *
+ * @param {String} comment
+ * @param {String} startMark
+ * @param {String} endMark
+ * @return {Void}
+ * @private
+ */
+function markPerf(comment, startMark, endMark) {
+  if (performance) {
+    performance.mark(endMark);
+    measure(comment, startMark, endMark);
```

Comment:
I believe this needs to be guarded in a `try` / `catch`. The rough `tldr;` is that anyone at any time can call `performance.clearMarks()` on us, and then calling `performance.measure(...)` here would throw an error.

Some additional context in https://github.com/qunitjs/qunit/pull/1319.

Reply:
Yup, this is being done here: https://github.com/ember-a11y/ember-a11y-testing/pull/115/files#diff-140c735f8ae8e94da2968b4bb2f97b7dR48, where the measure fn is defined 🙂

Reply:
Shoot. I'm sorry for not spotting that before, I thought I did search for where `measure` was coming from 😞

Reply:
No worries.